<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowBase</class>
 <widget class="QMainWindow" name="MainWindowBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fuel Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="vehicleLabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="entryFormTab">
       <attribute name="title">
        <string>Instructions</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>4</y>
          <width>611</width>
          <height>251</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1.  Click on settings-&gt;configure fuel manager
2.  Select a database
3.  Click on Vehicle.
4.  Type in a description
5.  Click apply
6.  Click ok
7.  Click file-&gt;select vehicle to choose a vehicle</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>340</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_Vehicle"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionConfigure_Fuel_Manager"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionExport_Data"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Fuel Manager</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>&amp;Login</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E, Return</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConfigure_Fuel_Manager">
   <property name="text">
    <string>&amp;Configure Fuel Manager...</string>
   </property>
  </action>
  <action name="actionSelect_Vehicle">
   <property name="text">
    <string>S&amp;elect Vehicle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExport_Data">
   <property name="text">
    <string>&amp;Export Data</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLogin</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConfigure_Fuel_Manager</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>configure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindowBase</receiver>
   <slot>updateAllData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_Vehicle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>selectVehicle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExport_Data</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>exportData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>openConfigureDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>login()</slot>
  <slot>configure()</slot>
  <slot>updateAllData()</slot>
  <slot>about()</slot>
  <slot>selectVehicle()</slot>
  <slot>exportData()</slot>
  <slot>openConfigureDialog()</slot>
 </slots>
</ui>
